internal/core/model.rs

```diff
@@ -1117,18 +1117,35 @@ impl<C: RepeatedComponent + 'static> Repeater<C> {
 pub struct StandardListViewItem {
     /// The text content of the item
     pub text: crate::SharedString,
+
+    /// If set to `true` the text can be edited.
+    pub editable: bool,
 }
 
-impl From<&str> for StandardListViewItem {
-    fn from(other: &str) -> Self {
-        return Self { text: other.into() };
+impl From<crate::SharedString> for StandardListViewItem {
+    fn from(value: crate::SharedString) -> Self {
+        StandardListViewItem { text: value, editable: false }
     }
 }
 
-impl From<SharedString> for StandardListViewItem {
-    fn from(other: SharedString) -> Self {
-        return Self { text: other };
-    }
+/// Represent an TableColumn header
+#[repr(C)]
+#[derive(Clone, Default, Debug, PartialEq)]
+pub struct TableColumn {
+    /// The title of the column header
+    pub title: crate::SharedString,
+
+    /// The minimum column width
+    pub min_width: f64,
+
+    /// The horizontal column stretch
+    pub horizontal_stretch: f32,
+
+    /// Sorts the column ascending
+    pub sort_by_ascending: bool,
+
+    /// Sorts the column descending
+    pub sort_by_descending: bool,
```

Comment:
I wonder if we should create an enum for that.
like `enum SortOrder{ None, Ascending, Descending  } `
What do you think?

Reply:
Yes sure sounds great.

Reply:
I was thinking the same. If that's technically possible then that would be a much nicer API, I think.

I suggest "Unsorted" instead of "None" for the variant where the items are unsorted.

What's the name of the property in `StandardListViewItem`? Maybe `sort-order`?